github/github.go

```diff
@@ -986,6 +996,59 @@ func (g *GitHub) UpdateReleasePage(
 	return release, nil
 }
 
+// toRepositoryRelease builds a repository release from the set of options
+func (urpo *UpdateReleasePageOptions) toRepositoryRelease() *github.RepositoryRelease {
+	request := &github.RepositoryRelease{}
+
+	if urpo.Name != "" {
+		request.Name = &urpo.Name
+	}
+	if urpo.Body != "" {
+		request.Body = &urpo.Body
+	}
+
+	request.Draft = &urpo.Draft
+	request.Prerelease = &urpo.Prerelease
+
+	if urpo.Latest {
+		request.MakeLatest = ptr.To("true")
+	} else {
+		request.MakeLatest = ptr.To("false")
+	}
```

Comment:
This can be a problem if you try to update the latest release without seeting the `Latest` parameter. Would the update call fail in that case because you're trying to remove the latest release parameter? Maybe it would be better to also make `upro.Latest` a bool pointer.

Reply:
If we do this, we should probably do the same for `Prerelease` and `Draft` parameters.

Reply:
I think I need to test this out. I feel like the semantics of `make_latest` suggest that you don't unset it being the latest when you set this to `"false"` (you just don't try to make it latest again), but ... this is a good point.

Reply:
> If we do this, we should probably do the same for Prerelease and Draft parameters.

Agreed on this part for sure.

Reply:
So I've tested behaviour of the API and I'm making all fields in the opts struct a pointer - because you can do partial updates of fields if you pass them, which means you might not want to set them at all.

For example, `{"body":"Description of the release","prerelease":true}` will write/update the description and set it to true, and subsequent calls with `{"body":"Updated description of the release"}` will not unset the pre-release value.